app/app.json

```diff
@@ -23,25 +23,14 @@
       },
       "bundleIdentifier": "com.dltx.idem",
       "supportsTablet": true,
-      "usesIcloudStorage": true,
-      "infoPlist": {
-        "NSPhotoLibraryUsageDescription": "IDEM accesses your photos to verify your claims",
-        "NSCameraUsageDescription": "Allow $(PRODUCT_NAME) to access your camera",
-        "NSMicrophoneUsageDescription": "Allow $(PRODUCT_NAME) to access your microphone"
-      }
+      "usesIcloudStorage": true
```

Comment:
Do we use icloud?

Reply:
yes, for expo-document-picker
ref : https://docs.expo.dev/versions/latest/sdk/document-picker/